scarb/tests/package.rs

```diff
@@ -458,6 +458,189 @@ fn workspace() {
         .name_and_version("workspace_dep", "1.0.0");
 }
 
+#[test]
```

Comment:
What would happen, if the package repo has a submodule? 🤔

Reply:
Could you elaborate on that question? I don't know if that's what you are asking for, but even if the package has submodule, the main one would be packaged.

Reply:
I just wondered whether we are taking into account changes of git submodules, but I see now that it would be reported by `git status -m`, so we should be good to go :)

Reply:
Yeah, at the end of the day it is one big repository. That's why we also discover repository outwards instead of just trying to look for it into the subpackage root, so that CWD doesn't matter when packaging.